distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,76 @@
+:title: Federated Identity
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated Sources during queries
+
+Each instance of {branding} may be configured with its own security policy that determines the
+resources a user may access and the actions they may perform. To decide whether a given request is
+permitted, {branding} references the SAML assertion stored internally in the requestor's
+<<_subject,Subject>>. This assertion is generated by the <<_security_token_service, STS>> during
+authentication and contains a collection of attributes that identify the requestor. Based on these
+attributes and the configured policy, {branding} makes an authorization decision. See
+<<_security_pdp, Security PDP>> for more information.
+
+This authorization process works when the requestor authenticates directly with {branding} as they
+are guaranteed to have a Subject. However, when federating, {branding} proxies requests to federated
+Sources and this poses a problem. The requestor doesn't authenticate with federated Sources, but
+Sources still need to make authorization decisions.
+
+To solve this problem, {branding} uses federated identity. When performing any federated request
+(query, resource retrival, etc), {branding} attaches the requestor's SAML assertion to the outgoing
+request. The federated Source extracts the assertion and validates its signature to make sure it
+was generated by a trusted entity. If so, the federated Source will construct a Subject for the
+requestor and perform the request using that Subject. The Source can then make authorization
+decisions using the process already described.
+
+How {branding} attaches SAML assertions to federated requests depends on the endpoint used to
```

Comment:
❓ Should we specify which sources do or do not support federated identity somewhere?

Reply:
I put a warning at the end that says federated identity support was added in DDF 2.8.x. Is that what you mean?